tracer/src/Datadog.Trace/ExtensionMethods/StringExtensions.cs

```diff
@@ -129,5 +129,70 @@ public static bool TryConvertToNormalizedTagName(this string value, bool normali
             normalizedTagName = StringBuilderCache.GetStringAndRelease(sb);
             return true;
         }
+
+        /// <summary>
+        /// Compact whitespaces in a string
+        /// https://github.com/DataDog/datadog-agent/blob/0454961e636342c9fbab9e561e6346ae804679a9/pkg/trace/obfuscate/obfuscate.go#L118-L146
+        /// </summary>
+        /// <param name="value">String instance</param>
+        /// <returns>Whitespace compacted string instance</returns>
+        public static string CompactWhiteSpace(this string value)
```

Comment:
Is this being used anywhere in this PR?

Reply:
Nice catch, no, I left these when trying to port the obfuscator, will remove it.

Reply:
done in [e68308a](https://github.com/DataDog/dd-trace-dotnet/pull/2402/commits/e68308a82bdee276bc6e8f7124e77362d5839bf0)